.gitignore

```diff
@@ -77,3 +77,6 @@ buck-out/
 
 # @end expo-cli
 demo.dev.monk.yml
+
+# vscode
+.vscode
```

Comment:
We could commit the vscode folder for other vscode devs, but yeah, we are a WS team.